toolkit/resources/manifests/package/license_file_exceptions.json

```diff
@@ -0,0 +1,37 @@
+{
+    "_comment1": "This file is used to allow specific files to be omitted from the license check process.",
+    "_comment2": "Each PkgException entry is a {'PackageName' and 'IgnoredFilesRegexList'}",
+    "_comment3": "The 'PackageName' is the name of the (sub)package to which the exception applies.",
+    "_comment4": "The 'IgnoredFilesRegexList' is a list of regular expressions that match files to be omitted from the license check.",
+    "_comment5": "The 'GlobalExceptionsRegexList' is a list of regular expressions that match all packages.",
+
+    "PkgExceptions": [
+        {
+            "PackageName": "gcc",
+            "IgnoredFilesRegexList": [
+                "^/usr/share/man/man7/gpl.7.gz$"
```

Comment:
I struggle to understand how these files would be caught up in the regexes listed in FuzzyLicenseNamesRegexList, etc. from license_file_names.json?

Reply:
`(?i).*gpl.*` is the one flagging this file.